cc3d/player5/Simulation/main_loops.py

```diff
@@ -99,6 +99,9 @@ def main_loop_player(sim, simthread=None, steppable_registry=None):
 
         cur_step += 1
 
+        if cur_step in pg.pause_at:
```

Comment:
good point. I was worried too but I know that unless you really want to put effort into it the list search cost is negligible. Still it might be wort exploring better option. Let me play with it

Reply:
@tjsego Fixed the issue

Reply:
Very cool. Is `str_to_int_container` somewhere available in the main repo? I don't see it in a PR, but maybe I missed it.

Reply:
Yes it is in the CompuCell3D repo (the corte one)